dotnet/dotnet-guestbook/.readmes/vscode/README.md

```diff
@@ -31,44 +31,52 @@ The Guestbook sample demonstrates how to deploy a Kubernetes application with a
+2. Click '+' next to the **Clusters** header to create a new cluster. If prompted, follow the instructions to log in to Google Cloud Platform.  
 
-4.  To stop the application, click the stop icon on the Debug Toolbar.
+3. Choose **Minikube** and then select **minikube**. 
+
+2. Select **Start**. Cloud Code will initiate a minikube cluster.
+
+#### Google Kubernetes Engine
+
+1. Navigate to the **Cloud Code - Kubernetes** explorer using the left side taskbar.
 
-### Run the app remotely on Google Kubernetes Engine
+2. Click '+' next to the **Clusters** header to create a new cluster. If prompted, follow the instructions to log in to Google Cloud Platform.  
 
-#### Set up a GKE cluster
+3. Choose **Google Kubernetes Engine**. If you have existing clusters associated with your GCP project, you can select one from this dialog. To create a new cluster, click **+ Create a new GKE Cluster** and follow these steps:
 
-1. Navigate to the Cloud Code - Kubernetes explorer using the left side taskbar.
+    a. Choose **Standard** or **Autopilot**. For more information, see [Comparing Autopilot and Standard modes](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview#comparison?utm_source=ext&utm_medium=partner&utm_campaign=CDR_kri_gcp_cloudcodereadmes_012521&utm_content=-).
 
-2. In the Google Kubernetes Engine Explorer, click '+' to create a new Google Kubernetes Engine cluster. If prompted, follow the instructions to log in to Google Cloud Platform.  
-![image](./img/kubernetes-explorer.png)
+    b. If prompted, click **Open** to permit Cloud Code to open the Cloud Console.
 
-3. Configure your new cluster using the GKE wizard and click 'Create Cluster'. Once your GKE cluster has been created, it will be displayed in the Google Kubernetes Engine Explorer.  
-![image](./img/gke-cluster-config.png)
+    c. In the Cloud Console, choose any configuration options that you want to customize and then click **Create**.
 
-4. Once your GKE cluster has been created, it will be displayed in the Google Kubernetes Engine Explorer.
+    d. Once the cluster has finished being created, return to VS Code and click **Refresh**.
 
-5. Your new cluster will be set as the active cluster by default. To switch clusters, right click on a different cluster in the GKE Explorer and select “Set Active Cluster”.  
-![image](./img/kubernetes-explorer-new-cluster.png)
+    You can now select your newly created cluster.
 
-#### Deploy app to GKE
+8. The cluster you select will be added to the **Clusters** tab in the **Kubernetes Explorer** and set as the default context. You can inspect the cluster's properties, make changes to the cluster's resources, and view logs by clicking the dropdown arrow next to the cluster name in the **Clusters** tab.
 
-1. Click on the Cloud Code status bar and select ‘Run on Kubernetes’.
+#### Deploy app to cluster
+
+1. Open the Command Palette (press Ctrl/Cmd+Shift+P) and then run the **Cloud Code: Run on Kubernetes** command.
```

Comment:
Instead of guiding users to Command Palette to run/debug an app, we want users to access these functionality via 'Development Sessions' explorer.

Can we use instructions similar to below to achieve this?
1. Navigate to the **Development Sessions** explorer in the **Cloud Code - Kubernetes** sidebar using the Activity bar on the left side of the IDE.
2. Click 'Play' icon in the title bar of the **Development Sessions** explorer to run the application.

Reply:
Done!

The [cloud.google.com](https://cloud.google.com/code/docs/vscode/running-an-application#running_your_application) instructions still use the Command Palette, should we update these to use the Development Sessions explorer as well?